apache-tomcat-mixin/dashboards/apache-tomcat-overview.libsonnet

```diff
@@ -160,7 +164,7 @@ local cpuUsagePanel = {
           },
         ],
       },
-      unit: 'percentunit',
+      unit: 'percent',
```

Comment:
was this change necessary? just checking

Reply:
@StefanKurek When I was comparing the two mixins side-by-side, I saw that this was `percent` in the other mixin and `percentunit` in the main one, I wasn't certain whether or not this change was necessary, but it felt right. Do you know what `percentunit` is and is it different from `percent`? I was basically just guessing that it was a typo or something.

Reply:
@algchoo i "think" one is for a raw percent and one is for a number 0-1 which will then show as a percent. so next i would ask if the original caused percentages that seemed too large (bigger than 100). that would be a sign that you need to switch.

Reply:
@StefanKurek Just tested this out and it probably definitely needs to be `percentunit`, thanks for calling this out 🤠